pkgs/applications/graphics/inkscape/test-ps2pdf-plugin.nix

```diff
@@ -0,0 +1,58 @@
+{ inkscape, runCommand, writeTextFile }:
+
+let
+  svg_file = writeTextFile {
+    name = "test.svg";
+    text = ''
+      <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+      <svg
+        width="100"
+        height="100"
+        viewBox="0 0 26.458333 26.458333"
+        version="1.1"
+        id="svg1"
+        sodipodi:docname="test.svg"
+        xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:svg="http://www.w3.org/2000/svg">
+        <defs
+          id="defs1" />
+        <g
+          id="layer1">
+          <text
+            xml:space="preserve"
+            style="font-size:3.175px;font-family:'noto sans';-inkscape-font-specification:'noto sans';fill:#f3102a;stroke:#000000;stroke-width:0.264583"
+            x="8.5666199"
+            y="6.2798061"
+            id="text1"><tspan
+              sodipodi:role="line"
+              style="stroke-width:0.264583"
+              x="8.5666199"
+              y="6.2798061"
+              id="tspan2">hallo</tspan></text>
+          <ellipse
+            style="fill:#f3102a;stroke:#000000;stroke-width:0.264583"
+            id="path3"
+            cx="13.475231"
+            cy="10.701576"
+            rx="3.282829"
+            ry="2.9031823" />
+        </g>
+      </svg>'';
+  };
+in
+runCommand "inkscape-test-eps"
+{
+  nativeBuildInputs = [ inkscape ];
+} ''
+  echo ps test
+  inkscape ${svg_file} --export-type=ps -o test.ps
+  inkscape test.ps -o -
+
+  echo eps test
+
+  inkscape ${svg_file} --export-type=eps -o test.eps
+  inkscape test.eps -o -
+
+  touch $out
```

Comment:
This builds even without the patch.

Reply:
Thanks you very much, oversaw that inkscape does not return an error code, ...